From 204a91fbb00c6419cce05a0b3fa44ec8d002ddaa Mon Sep 17 00:00:00 2001
From: yinbo.zhu <yinbo.zhu@nxp.com>
Date: Wed, 10 May 2017 19:44:55 +0800
Subject: [PATCH 071/211] uboot: Kconfig: add ERRATUM config to Kconfig for solve compile issue.

add erratum config to Kconfig and instead of config.h's config
for solve ls2080rdb compiling issue

Signed-off-by: Sriram Dash <sriram.dash@nxp.com>
Signed-off-by: yinbo.zhu <yinbo.zhu@nxp.com>
---
 .../include/asm/arch-fsl-layerscape/immap_lsch3.h  |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/include/asm/arch-fsl-layerscape/immap_lsch3.h b/arch/arm/include/asm/arch-fsl-layerscape/immap_lsch3.h
index 2409879..4a51e55 100644
--- a/arch/arm/include/asm/arch-fsl-layerscape/immap_lsch3.h
+++ b/arch/arm/include/asm/arch-fsl-layerscape/immap_lsch3.h
@@ -123,6 +123,7 @@
 #define DCFG_RCWSR15_IFCGRPABASE_QSPI	0x3
 
 #define DCFG_DCSR_BASE		0X700100000ULL
+#define DCSR_BASE              0x700000000ULL
 #define DCSR_USB_PHY1                  0x4600000
 #define DCSR_USB_PHY2                  0x4610000
 #define DCSR_USB_PHY_RX_OVRD_IN_HI     0x1006
-- 
1.7.1

